docs/getting-started/uwp.rst

```diff
@@ -166,3 +162,33 @@ You can now run the application to see it in action.
 .. image:: uwp/_static/create.png
 
 .. image:: uwp/_static/list.png
+
+Generate runtime directives
+---------------------------
+
```

Comment:
In fact, I would phrase this section as "stuff you need to know for deployment" rather than making a step in the step-by-step walkthough